pages/index.tsx

```diff
@@ -162,14 +162,14 @@ export const HomePage: NextPage = () => {
         <CreateListAlert selectedService={selectedResource} />
         <CreateListDrawer />
       </CreateListProvider>
-      <Box px="112px" py="96px">
+      <Box px="10%" py="96px" minWidth="800px">
```

Comment:
Just curious, any particular reason for the 10% x-padding and 800px min width? Screenshots look good but numbers seem kinda random (tho I guess that's just kinda how UI design goes?).

Could use [Chakra's `useTheme()` hook](https://chakra-ui.com/docs/hooks/use-theme) and use one of the [default breakpoints](https://chakra-ui.com/docs/styled-system/theme#breakpoints) to avoid hard-coding this min width everywhere tho how their defaults are decided are also a mystery to me.